ext/ffi/ffi.c

```diff
@@ -2926,22 +2926,11 @@ static zend_function *zend_ffi_get_func(zend_object **obj, zend_string *name, co
 	zend_function   *func;
 	zend_ffi_type   *type;
 
-	if (ZSTR_LEN(name) == sizeof("new") -1
-	 && (ZSTR_VAL(name)[0] == 'n' || ZSTR_VAL(name)[0] == 'N')
-	 && (ZSTR_VAL(name)[1] == 'e' || ZSTR_VAL(name)[1] == 'E')
-	 && (ZSTR_VAL(name)[2] == 'w' || ZSTR_VAL(name)[2] == 'W')) {
+	if (zend_string_equals_literal_ci(name, "new")) {
```

Comment:
zend_string_equals_literal_ci() is going to be slower. I would prefer to revert this.